cpp/oneapi/dal/algo/basic_statistics/backend/gpu/compute_kernel_dense_impl_dpc.cpp

```diff
@@ -1209,7 +1081,7 @@ compute_kernel_dense_impl<Float, List>::compute_by_blocks(const pr::ndview<Float
     DECLSET_IF(Float*, asum2cent_ptr, sum2cent_based_stat, ndbuf.get_sum2cent().get_mutable_data())
 
     const auto* data_ptr = data.get_data();
-    const auto wg_size = be::device_max_wg_size(this->q_);
```

Comment:
I agree with removing `this->`, but I have question about the variable name. 
http://oneapi-src.github.io/oneDAL/contribution/coding_guide.html

`q_` is wrong for many reasons. First, the rule about private members says
> `private` class members and methods should have an additional underscore (_) at the beginning

... and not at the end.

And then there are also rules about variable names 

* Give descriptive and meaningful names.
* Avoid ambiguity.
* Do not abbriviate (sic) words.
* Do not remove letters from words

`q_` violates all of these rules.

I see that this is a more global problem because it's also done in other files. Would it make sense to refactor this?

Reply:
I also agree with this point, that in different algorithms we have different names for common variables, like queues or communicators, and it is a global problem. But, my view is that we need to do it consistency in additional pr for each algo. For LOM I will fix it in new commit. Thanks!